src/runtime/policy/canonical-policy-annotations.ts

```diff
@@ -29,7 +29,7 @@ annotation allowedRetention(medium: Text, encryption: Boolean)
 annotation maxAge(age: Text)
   targets: [PolicyTarget]
   retention: Source
-  doc: 'Indicates the maximum age the target data can be in order to be used. Age is a string like "2 days". Supported units are minutes, hours, days.'
+  doc: 'Indicates the maximum age the target data can be in order to be used. Age is a string like "2d". Supported units are minutes ("m"), hours ("h"), days ("d").'
```

Comment:
fyi: I plan to migrate TTLs from the d/m/h to days/minutes/hours, but just didn't want to make the switch to new capabilities a breaking change. I should actually just add support for both to avoid needing to migrate policies later. I'll try to send the change today.

Reply:
Supporting both makes sense to me. I'm thinking I'll just keep this as it is, and then if we decide we prefer one over the other we can migrate everything later on.